lib/graphql-hive/usage_reporter.rb

```diff
@@ -16,14 +17,11 @@ def self.instance
 
       def initialize(options, client)
         @@instance = self
-
         @options = options
         @client = client
-
         @options_mutex = Mutex.new
-        @queue = Queue.new
-
         @sampler = Sampler.new(options[:collect_usage_sampling], options[:logger]) # NOTE: logs for deprecated field
+        @queue = BoundedQueue.new(bound: options[:buffer_size], logger: options[:logger])
```

Comment:
i know that the `bounded_queue_multiple` config was removed, but should we still have the bound as the buffer size * 5?
```suggestion
        @queue = BoundedQueue.new(bound: options[:buffer_size].to_i * 5, logger: options[:logger])
```

Reply:
We had a discussion here about this! https://github.com/rperryng/graphql-ruby-hive/pull/38#discussion_r1821310282

We don't need this to be considerably larger than the buffer size because the thread wakes up for **every single operation**, and calls `pop`. 

So ideally, this won't every reach larger than size of `~1`. But to be safe, we can provide a larger queue (buffer size). If we reach this limit, we have other things that are going wrong.

Reply:
that makes sense! i thought we still wanted it to be considerably larger. thanks for clearing that up!